src/Yaapii.Atoms/Text/TextBase64.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Yaapii.Atoms.Bytes;
+using Yaapii.Atoms.IO;
+
+namespace Yaapii.Atoms.Text
+{
+    public sealed class TextBase64 : IText
+    {
+        private readonly IText origin;
+
+        /// <summary>
+        /// A <see cref="string"/> as string <see cref="IText"/>
```

Comment:
Comment makes no sense. Do you mean A string as Base64 encoded text?

Reply:
@koeeenig 
fixed